src/ucx_rma_plugin.c

```diff
@@ -425,22 +420,28 @@ static ucs_status_t nccl_ucx_rma_am_rkey_cb(void *arg, void *data, size_t length
 {
   nccl_ucx_rma_send_comm_t *comm     = (nccl_ucx_rma_send_comm_t*)arg;
   nccl_ucx_rma_rkey_buf_t  *rkey_buf = (nccl_ucx_rma_rkey_buf_t*)data;
+  ucs_status_t status;
   
   if (comm->rkeys[rkey_buf->index].rkey) {
     ucp_rkey_destroy(comm->rkeys[rkey_buf->index].rkey);
   }
   comm->rkeys[rkey_buf->index].id = rkey_buf->id;
-  UCXCHECK(ucp_ep_rkey_unpack(comm->ep, rkey_buf->buf,
-                              &comm->rkeys[rkey_buf->index].rkey));
-  return UCS_OK;
+  status = ucp_ep_rkey_unpack(comm->ep, rkey_buf->buf,
+                              &comm->rkeys[rkey_buf->index].rkey);
+  if (status != UCS_OK) {
+    WARN("Failed: UCX am rkey cb: rkey unpack error %s",
+         ucs_status_string(status));
+  }
+
+  return status;
```

Comment:
According to UCP AM API, AM callback maybe return either `UCS_OK` or `UCS_INPROGRESS`.
The former means that data is no longer needed and can be released, the latter means that data will need to be accessed later and should not be released.
So, need to return `UCS_OK` in any case here

Reply:
fixed with new behavior